### PR TITLE
Feature request feature

### PR DESCRIPTION
**Pull request by BradWells**
I used (self, ctx, *, text) so hopefully it looks like you can send text.
Using github editor cause lazy

### CODE REVIEW

**Comment by BradWells** [comment]
Phase 2 would be random insults, which has good synergy with !ban

File: cogs/randomStuff.py [hunk 1]
```diff
@@ -143,6 +143,12 @@ async def reactions(self, ctx):
     async def ban(self, ctx, name):
         await ctx.send(f'Shut up, {name.title()}')
 
+    @commands.command(name='featurerequest', aliases=['request'])
+    async def feature_request(self, ctx, *, text):
+        """Request a new feature for the discord bot!"""
+        #TODO random insults
+        await ctx.send('Add it yourself, bozo! https://github.com/Joelpls/discord-bot')
```
Comment (iantrich) [inline cogs/randomStuff.py]: Use https://github.com/Joelpls/discord-bot/issues/new?assignees=&labels=&template=feature_request.md&title=

Comment (BradWells) [reply cogs/randomStuff.py]: I think you sent the wrong link, that doesn't look like an insult library

Comment (Joelpls) [reply cogs/randomStuff.py]: ya make it go to this link, @BradWells

**Comment by BradWells** [comment]
Ok please check again

**Review by Joelpls — APPROVED** [review]

**Pull request merged**